app/frontend/src/features/ReportButton.tsx

```diff
@@ -0,0 +1,246 @@
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "components/Dialog";
+import { BugIcon } from "components/Icons";
+import Snackbar from "components/Snackbar";
+import TextField from "components/TextField";
+import { useAuthContext } from "features/auth/AuthProvider";
+import {
+  BUG_DESCRIPTION_HELPER,
+  BUG_DESCRIPTION_NAME,
+  BUG_REPORT_SUCCESS,
+  CANCEL,
+  EXPECT_HELPER,
+  EXPECT_NAME,
+  PROBLEM_HELPER,
+  PROBLEM_NAME,
+  REPORT,
+  REPORT_BUG_BUTTON,
+  REPORT_CONTENT_BUTTON,
+  REPORT_CONTENT_EMAIL,
+  REPORT_CONTENT_MESSAGE,
+  SUBMIT,
+  WARNING,
+} from "features/constants";
+import { Error as GrpcError } from "grpc-web";
+import { ReportBugRes } from "proto/bugs_pb";
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { useMutation } from "react-query";
+import { service } from "service";
+import makeStyles from "utils/makeStyles";
+
+export interface BugReportFormData {
+  subject: string;
+  description: string;
+  results: string;
+}
+
+const useStyles = makeStyles((theme) => ({
+  button: {
+    "&:hover": {
+      backgroundColor: darken(theme.palette.error.main, 0.1),
+    },
+    backgroundColor: theme.palette.error.main,
+  },
+  typeButton: {
+    display: "block",
+    margin: "0 auto",
+    "& + &": {
+      marginBlockStart: theme.spacing(2),
+    },
+  },
+  field: {
+    "& + &": {
+      marginBlockStart: theme.spacing(2),
+    },
+  },
+  startIcon: {
+    [theme.breakpoints.down("sm")]: {
+      margin: 0,
+    },
+  },
+}));
+
+export default function ReportButton({
+  isResponsive = true,
+}: {
+  isResponsive?: boolean;
+}) {
+  const theme = useTheme();
+  const isBelowMd = useMediaQuery(theme.breakpoints.down("sm"));
+
+  const classes = useStyles();
+  const [isOpen, setIsOpen] = useState(false);
+  const [type, setType] = useState<"initial" | "bug" | "content">("initial");
+  const {
+    register,
+    handleSubmit,
+    reset: resetForm,
+  } = useForm<BugReportFormData>();
+  const userId = useAuthContext().authState.userId;
+  const {
+    data: bug,
+    error,
+    isLoading,
+    mutate: reportBug,
+    reset: resetMutation,
+  } = useMutation<ReportBugRes.AsObject, GrpcError, BugReportFormData>(
+    (formData) => service.bugs.reportBug(formData, userId),
+    {
+      onSuccess: () => {
+        setIsOpen(false);
+      },
+    }
+  );
+
+  const handleClose = () => {
+    resetForm();
+    resetMutation();
+    setIsOpen(false);
+    setTimeout(
+      () => setType("initial"),
+      theme.transitions.duration.leavingScreen
+    );
+  };
+
+  const onSubmit = handleSubmit((data) => {
+    reportBug(data);
+  });
+
+  return (
+    <>
+      {bug && (
+        <Snackbar severity="success">
+          <>
+            {BUG_REPORT_SUCCESS}
+            <Link href={bug.bugUrl}>{bug.bugId}</Link>.
+          </>
+        </Snackbar>
+      )}
+      <Button
+        aria-label="Report a bug"
```

Comment:
```suggestion
        aria-label={REPORT}
```
For small screens so it matches the label we hide and shrinks to the icon only. I'm guessing the label of that button must have changed in between...